sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/optimizer/ComputeCurrentTimeSuite.scala

```diff
@@ -163,6 +178,16 @@ class ComputeCurrentTimeSuite extends PlanTest {
     checkLiterals({ zoneId: String => CurrentDate(Some(zoneId)) }, numTimezones)
   }
 
+  test("datetime function CurrentDate and LocalTime are Unevaluable") {
```

Comment:
This does not test the optimizer rule and we should keep it in `DateExpressionsSuite`